zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -25,7 +25,7 @@ pub const N_PHASE1_COLUMNS: usize =
 // Number of copy columns
 pub const N_COPY_COLUMNS: usize = 2;
 
-pub const N_BYTE_LOOKUPS: usize = 24;
+pub const N_BYTE_LOOKUPS: usize = 28;
```

Comment:
Where this 4 more lookups come from?

Reply:
More words are allocated in the end_tx step, so more data needs to be stored in these steps. So either the circuit can be made wider with more capacity for byte values (what this change does) or we just allow the step to use more rows (currently limited by `MAX_STEP_HEIGHT`) to store all the necessary data.

Reply:
So, as we prefer height over width, giving more space to MAX_STEP_HEIGHT makes more sense, is it?

Reply:
Generall yes. And in this specific case especially true because increasing the max height just allows the end_tx to use more rows, which doesn't impact the cost of the other opcodes. Making the circuit wider or increasing the number of lookups also impacts the cost of the other opcodes because that cost is for each row, regardless of what opcode is being processed.

Reply:
> Where this 4 more lookups come from?

I test the number one by one, because I need more `Word<F>(RandomLinearCombination)`
1. tx_gas_tip_cap, tx_gas_fee_cap, base_fee_plus_tip = 3* 32  cells
2. mul_base_fee_by_gas_used = 32 + 8 cells
3. treasury updatebalance = 32 * 2 cells
4. and two AddWords = 2 cells

sum ≈ 200+

our MAX_STEP_HEIGHT equals to 21, so we need more 10 columns, but N_BYTE_LOOKUPS has some remaining
so I test to increase columns one by one

Reply:
Yeah, I think it makes sense to increase the MAX_STEP_HEIGHT